DP3TApp/Screens/Vaccination/NSVaccinationInfoViewController.swift

```diff
@@ -90,36 +92,60 @@ private class VaccinationInfoContentView: NSSimpleModuleBaseView {
     // MARK: - Setup
 
     private func setup() {
+        let bookNowLabel = NSLabel(.textBold)
+        bookNowLabel.text = "vaccination_impf_check_title".ub_localized
```

Comment:
this should be 
config?.vaccinationBookingInfo.value?.impfcheckTitle ?? "vaccination_impf_check_title".ub_localized